### PR TITLE
Try out COCO directly for image requests

### PR DESCRIPTION
**Pull request by matthew-andrews**

### CODE REVIEW

**Comment by catalin-ilea** [comment]
lgtm

**Comment by Nesze** [comment]
+1

File: server/lib/fetch-capi-json.js [hunk 1]
```diff
@@ -4,7 +4,8 @@ module.exports = (endpoint) => {
 	return fetch(endpoint, {
 		timeout: 9000,
 		headers: {
-			'X-Api-Key': process.env.API_KEY
+			'X-Api-Key': process.env.API_KEY,
+			'X-Policy': 'INCLUDE_RICH_CONTENT, INCLUDE_COMMENTS, INTERNAL_UNSTABLE'
```
Comment (matthew-andrews) [inline server/lib/fetch-capi-json.js]: i forgot the authorization header

Comment (catalin-ilea) [inline server/lib/fetch-capi-json.js]: It will work without auth for now...but good point, better add it now

**Comment by kovacshuni** [comment]
+1

**Pull request merged**